.github/workflows/go.yml

```diff
@@ -0,0 +1,89 @@
+name: Go
+
+on:
+  push:
+    branches: [ "v2" ]
+    tags:
+      - "*"
+  pull_request:
+    branches: [ "v2" ]
+
+
+jobs:
+  configure:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.18
+        
+  upload:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - name: linux-x86_x64
+            os: linux
+            arch: amd64
+          - name: linux-arm64
+            os: linux
+            arch: arm64
+          - name: linux-arm32
+            os: linux
+            arch: arm
+          - name: windows-amd64
+            os: windows
+            arch: amd64
+          - name: windows-amd32
+            os: windows
+            arch: 386
+          - name: macos-intel64
+            os: darwin
+            arch: amd64
+          - name: macos-AppleSilicon
```

Comment:
Why not `macos-amd64` and `macos-arm64`?

Reply:
okay, ill change it